exercises/binary-search/src/example/java/BinarySearch.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.List;
+
+public class BinarySearch<T extends Comparable<T>> {
+
+    public static String ARRAY_MUST_BE_SORTED = "Array should be sorted.";
```

Comment:
Hello, Stuart, thank you for the answer. So the takeaway is that comments may be used only in the example implementations, not the tests. Am I right?